protocols/raft/src/main/java/io/atomix/protocols/raft/roles/LeaderRole.java

```diff
@@ -1465,10 +1464,12 @@ private void replicate(Indexed<ZeebeEntry> indexed) {
               // up to date with the latest entries so it can handle configuration and initial
               // entries properly on fail over
               if (commitError == null) {
+                result.complete(indexed);
```

Comment:
What we do when this future is complete? Or more specific what are the consequences of this change now?

Reply:
Prior, the append would complete when the entry was written to the log; now it's completed when the entry is committed, meaning it's been replicated to a majority quorum. This is the same behaviour as what we currently have and what the backpressure in the log storage appended expects.

Reply:
makes sense thanks